src/main/java/uk/gov/hmcts/reform/civil/helpers/BundleRequestMapper.java

```diff
@@ -152,18 +152,19 @@ private BundlingCaseData mapRespondent2Applicant2Details(BundlingCaseData bundli
 
     private ServedDocument mapServedDocuments(ServedDocumentFiles servedDocumentFiles) {
         List<BundlingRequestDocument> bundlingServedDocFiles = new ArrayList<>();
-        if (!Optional.ofNullable(servedDocumentFiles).isEmpty() && null != servedDocumentFiles.getParticularsOfClaimDocument()) {
-            servedDocumentFiles.getParticularsOfClaimDocument().forEach(document -> {
-                bundlingServedDocFiles.add(BundlingRequestDocument.builder()
+        if (null == servedDocumentFiles || null == servedDocumentFiles.getParticularsOfClaimDocument()) {
+            return null;
```

Comment:
Wrong return value.

If you want to short-circuit this, you need to make sure that it will return what the previous implementation was returning, which in this instance is never null. This needs to be as follows:

```suggestion
            return  ServedDocument.builder().particularsOfClaimDocument(ElementUtils.wrapElements(new ArrayList<>())).build();
```